src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let accountingCount = 0;
+
+  return function(addition1) {
+    accountingCount++;
+
+    return function(addition2) {
+      return accountingCount > 3
+        && accountingCount % 2 === 0
+        ? 'Bzzz... Error!'
+        : addition1 + addition2;
```

Comment:
if line length >= 80, find another way, but do not format it this way
```suggestion
      return accountingCount > 3 && accountingCount % 2 === 0
        ? 'Bzzz... Error!'
        : addition1 + addition2;
```